modules/workflow-workflowoperation/src/main/java/org/opencastproject/workflow/handler/workflow/DuplicateEventWorkflowOperationHandler.java

```diff
@@ -227,7 +246,7 @@ public WorkflowOperationResult start(final WorkflowInstance workflowInstance, fi
 
     SeriesInformation series = null;
     AccessControlList seriesAccessControl = null;
-    if (!seriesId.isEmpty()) {
+    if (!seriesId.isEmpty() && !seriesId.startsWith("${")) {
```

Comment:
This doesn't check that the substitution terminates.  Is that expected?

Reply:
fixed